src/components/PromoSheet/index.tsx

```diff
@@ -1,64 +1,72 @@
-import lang from 'i18n-js';
 import React, { useCallback, useEffect, useReducer } from 'react';
```

Comment:
what does moving this to an index accomplish? why not just delete `/promos` but keep the file name as `PromoSheet.tsx`? just curious

Reply:
Ah, originally I had the Row component as a separate file in the dir. Since it's all in one file now there's not strong reason for it atm.